microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -2264,6 +2264,69 @@ Receiver:
 ---
 
 
+<!-- ====================================================================== -->
+## Incoming from RelNotes 113 (todo)
+
+
+<!-- ------------------------------ -->
```

Comment:
@aluhrs13 @novac42 - what section of the Outline (see [PR main page](https://github.com/MicrosoftDocs/edge-developer/pull/2585)) should this "File API" section go in?